src/fuse/mod.rs

```diff
@@ -151,6 +151,16 @@ pub struct FsSetxattrParam<'a> {
     pub position: u32,
 }
 
+/// Param passed to setxattr
+#[derive(Debug)]
+pub struct FsReleaseParam {
```

Comment:
Add document for each field. 
Refer to https://github.com/datenlord/sync_fuse/commit/6f68fa406c644c1d7473f296ccd0b94e5f070722

Reply:
Done.